src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectPaperTableRepo.kt

```diff
@@ -285,4 +299,19 @@ class ProjectPaperTableRepo(
             it[ProjectPaperTable.decision] = decision
         }
     }
+
+    override suspend fun getLaterProjectPapers(projectId: UUID, localPaperId: Long, stage: Long): List<ProjectPaper> =
+        db.query {
+            ProjectPaperTable
+                .selectAll()
+                .where {
+                    (ProjectPaperTable.projectId eq projectId) and
+                        (
+                            (ProjectPaperTable.stage eq stage and (ProjectPaperTable.localPaperId greater localPaperId))
+                                or (ProjectPaperTable.stage greater stage)
+                            )
```

Comment:
Let's store the OPs so that the where predicate is easier to read.

```suggestion
        db.query {
            val sameStageButGreaterLocalIdOp =
                (ProjectPaperTable.stage eq stage) and (ProjectPaperTable.localPaperId greater localPaperId)
            val greaterStageOp = ProjectPaperTable.stage greater stage

            ProjectPaperTable
                .selectAll()
                .where {
                    (ProjectPaperTable.projectId eq projectId) and (sameStageButGreaterLocalIdOp or greaterStageOp)
```

Reply:
Great idea. Thanks.